tools/cp-utility/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "cp-utility"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Is there a reason for 2021 here?

Reply:
https://doc.rust-lang.org/edition-guide/editions/index.html
2021 is the latest edition.